src/pages/admin/categories/AddEditCategory/index.js

```diff
@@ -224,14 +226,14 @@ const AddEditCategory = () => {
                   </Form.Control.Feedback>
                 </Form.Group>
                 <Button
-                  className={style.button}
+                  buttonStyle={style.button}
                   type="submit"
                   disabled={submitStatus}
-                >
-                  {loc.pathname === '/admin/add-category'
+                  buttonSize="def"
+                  buttonLabel={loc.pathname === '/admin/add-category'
                     ? 'Add Category'
                     : 'Save Category'}
```

Comment:
you can put this in a function and return the string value of 'Add Category' or 'Save Category' instead of doing it inline